src/EnergyPlus/ConductionTransferFunctionCalc.cc

```diff
@@ -385,6 +387,10 @@ namespace ConductionTransferFunctionCalc {
 						}
 					}
 				}
+				if ( Material( CurrentLayer ).Thickness > 3.0 ) {
+					ShowWarningError( "InitConductionTransferFunctions: Found Material that is possibly too thick for CTF calculation" );
```

Comment:
I vaguely remember that the 3 m limit is there because of serious problems in CTF's FD.  I worry about letting it run with just a warning, whereas the previous IDD limit was a hard stop.  I think it would be safer to severe-> fatal here to avoid hard crashes when folks want to model an old castle or cave.